packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartAxisTooltip.stories.tsx

```diff
@@ -1,7 +1,17 @@
 import * as React from 'react';
 import { VerticalBarChart, IVerticalBarChartDataPoint } from '@fluentui/react-charts-preview';
-//import { Stack, TextField } from '@fluentui/react';
-import { Checkbox, Field, Radio, RadioGroup, CheckboxOnChangeData, CheckboxProps } from '@fluentui/react-components';
+import {
+  Checkbox,
+  Field,
+  Radio,
+  RadioGroup,
+  CheckboxOnChangeData,
+  CheckboxProps,
+  Input,
+  InputProps,
+  InputOnChangeData,
+} from '@fluentui/react-components';
+import { StackShim } from '@fluentui/react-migration-v8-v9';
```

Comment:
Use strictly v9 controls only

Reply:
In the components mapping, it is written Stackshim can be used in V9 instead of Stack in V8

Reply:
dont use stack. Use css grid layout or flex box if needed.
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Basic_concepts_of_grid_layout